docs/parity-testing.md

```diff
@@ -0,0 +1,261 @@
+         ...
+      },
+      "lambda_get_fn_2": {
+        ...
+        "Configuration": {
+          "FunctionName": "<function-name:2>",
+          "FunctionArn": "arn:aws:lambda:<region>:111111111111:function:<function-name:2>",
+          "Role": "arn:aws:iam::111111111111:role/<resource:1>",
+          ....
+        },
+      },
+
+    }
+  }
+}
+```
+
+## Tips and Tricks for Transformers
+
+Getting the transformations right can be a tricky task and we appreciate the time you spend on writing parity snapshot tests for LocalStack! We are aware that it might be challenging to implement transformers that work for AWS and LocalStack responses. 
+
+In general, we are interested in transformers that work for AWS. Therefore, we recommend also running the tests and testing the transformers against AWS itself. 
+
+Meaning, after you have executed the test with the `snapshot-update` flag and recorded the snapshot, you can run the test without the update flag against the `AWS_CLOUD` test target. If the test passes, we can be quite certain that the transformers work in general. Any deviations with LocalStack might be due to missing parity. 
+
+You do not have to fix any deviations right away, even though we would appreciate this very much! It is also possible to exclude the snapshot verification of single test cases, or specific json-pathes of the snapshot. 
+
+### Skipping verification of snapshot test
+
+Snapshot verification is enabled by default. If for some reason you want to skip any snapshot verification, you can set the parameter `--snapshot-skip-all`.
+
+If you want to skip verification for or a single test case, you can set the pytest marker `skip_snapshot_verify`. If you set the marker without a parameter, the verification will be skipped entirely for this test case.
+
+Additionally, you can exclude certain paths from the verification only.
+Simply include a list of json-paths. Those paths will then be excluded from the comparison:
+
+```python 
+@pytest.mark.skip_snapshot_verify(
+        paths=["$..LogResult", "$..Payload.context.memory_limit_in_mb"]
+    )
+    def test_something_that_does_not_work_completly_yet(self, lambda_client, snapshot):
+        snapshot.add_transformer(snapshot.transform.lambda_api())
+        result = lambda_client....
+        snapshot.match("invoke-result", result)
+```
+
+{{< alert title="Note">}}
+Generally, [transformers](#using-transformers) should be used wherever possible to make responses comparable. 
+If specific paths are skipped from the verification, it means LocalStack does not have parity yet.
+{{< /alert >}}
+
+### Debugging the Transformers
+
+Sometimes different transformers might interfere, especially regex transformers and reference transformations can be tricky We added debug logs so that each replacement step should be visible in the output to help locate any unexpected behavior. You can enable the debug logs by setting the env `DEBUG_SNAPSHOT=1`.
+
+```json 
```

Comment:
This isn't json, which (I guess) breaks the rendering here in a weird way:
![image](https://github.com/localstack/localstack/assets/2796604/cc4b226a-e050-44d0-8f50-ed7ed89e787b)